github/filecoin-project.yml

```diff
@@ -3590,16 +3504,16 @@ repositories:
     advanced_security: false
     allow_update_branch: false
     archived: false
-    collaborators:
-      admin:
-        - vmx
     has_discussions: false
     merge_commit_message: PR_TITLE
     merge_commit_title: MERGE_MESSAGE
     secret_scanning_push_protection: true
     secret_scanning: true
     squash_merge_commit_message: COMMIT_MESSAGES
     squash_merge_commit_title: COMMIT_OR_PR_TITLE
+    teams:
```

Comment:
I'm all for keeping the permissions minimal. I just want to point out that here the `proofs-crate-owners` don't have admin access. I don't know if that's something desired for symmetry with other repos.